### PR TITLE
Add missed generated module

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description
Add missed generated module back to GeneratedModuleList.txt

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi Beisi,
I compared the file with generation branch and found some differences. I think we need to make sure they are alligned
<- Left: this PR                      -> Right: generation branch
image

File: tools/GeneratedModuleList.txt [hunk 1]
```diff
@@ -32,6 +34,7 @@ Az.ProviderHub
 Az.RedisEnterpriseCache
 Az.ResourceMover
 Az.SpringCloud
+Az.StackHCI
```
Comment (BethanyZhou) [inline tools/GeneratedModuleList.txt:37]:
```suggestion
```

**Comment by BethanyZhou** [comment]
Thanks @isra-fel ,

- Removed Az.StackHCI since it's a handcrafted module actually
- Hybrid modules should not be included in this list, so I will keep Az.ResourceGraph and Az.Websites on hold
- Added Az.ConnectedMachine to list

**Review by isra-fel — APPROVED** [review]
LGTM!

**Pull request merged**